.github/workflows/ci.yml

```diff
@@ -49,6 +53,10 @@ jobs:
           mkdir -p $NXF_SINGULARITY_CACHEDIR
           mkdir -p $NXF_SINGULARITY_LIBRARYDIR
 
+      - name: Fix Longranger repo
```

Comment:
https://github.com/sanger-tol/genomeassembly/blob/fixes_before_release/conf/test_github.config

Maybe it is easier to change the image for the process in the profile config file. 

This profile is only for Github, so you can change it here.